src/webui/backend/webui/api/oauth.py

```diff
@@ -35,8 +38,10 @@ def oauth_redirect():
     else:
         scope_role = ''
 
+    hostname = construct_snowflake_hostname(REGION, account)
```

Comment:
did you test this with WebUI? I think this is currently supported by a user entering the region to complete the URL in the login field, here —

> ![image](https://user-images.githubusercontent.com/42752788/98604502-9da2d600-2298-11eb-9836-6c5f4e0459da.png)

changing it like this will break that support, tho, and limit the UI to only work with a single region's accounts. the idea right now is that a single WebUI deployment should allow folks to log into two accounts in two different regions.

Reply:
Ah. I had not tried that. Will close this PR. Thank you.